bindings/rust-bindings/etc/chainList.toml

```diff
@@ -0,0 +1,197 @@
+[[chains]]
```

Comment:
Can we just use the existing `<repo-root>/chainList.toml` file instead of duplicating here?

Reply:
No we can't unfortunately because then it won't be included in the crates release and when someone uses the crate, it won't build because the file is missing. I will continue trying to figure out a work around but this is the best solution I found after a few hours

Reply:
Ok no worries. Doesn't need to block this pr, just a potential future cleanup